pkg/setting/provider.go

```diff
@@ -129,6 +131,10 @@ func (o *OSSImpl) Section(section string) Section {
 
 func (OSSImpl) RegisterReloadHandler(string, ReloadHandler) {}
 
+func (o OSSImpl) FeatureToggles() map[string]bool {
+	return o.Cfg.FeatureToggles
```

Comment:
If I'm not wrong, this is a bit risky because this exposes the settings internals and anyone could eventually modify them from outside (see [this example](https://play.golang.org/p/uanuKQtMXh1). So, we would need to, at least, return a copy.

That being said, now I'm wondering if we could start by reducing the exposed API and simply define a method that tells whether a given toggle is enabled or not (by just returning a boolean).

Any thoughts? 🤔  Thanks!